chinese/gui.py

```diff
@@ -75,24 +75,24 @@ def load_menu():
         )
 
     add_menu('Chinese::Bulk Fill')
-    add_menu_item('Chinese::Bulk Fill', _('Hanzi'), bulk_fill_hanzi)
```

Comment:
For these removals of the usage of _(), please note this change in [Anki](https://github.com/ankitects/anki/commit/1c5f94d46f00418de32c4057c28bdb5f9f4c7316#diff-b24794048d8be8242a9f8f89652a1456cd26f5451e1fcc8a7c5fcd98baff3fd6L158)
You will see that the function does nothing, and simply triggers a print to stdout to warn about its deprecation, so the removal should have no effect.